public/locales/pt/metrics.json

```diff
@@ -1,5 +1,4 @@
 {
-    "headerTitle": "Resumo das Avaliações",
-    "header":
-    "As estatísticas abaixo consideram a classificação mais votada afirmações individuais. Este discurso contém {{totalClaims}} afirmações, das quais {{totalClaimsReviewed}} receberam um total de {{total}} (100%) avaliações:"
+    "headerTitle": "Avaliações de afirmações",
+    "header": "Considerando todas as afirmações analisadas (100%)"
```

Comment:
Please confirm with the designers what "100%" means in this context, we shouldn't hard code this information but rather use real data from the backend.

Reply:
![image](https://user-images.githubusercontent.com/73478823/181388743-1e7722c4-4f7c-497c-b2d3-91d1afedbc3a.png)

Reply:
better omit 100% ?

Reply:
Yes, better to omit.